src/intrinsics/react-mocks/mocks.js

```diff
@@ -29,34 +75,126 @@ let reactCode = `
         return true;
```

Comment:
What is this function?

Reply:
(I guess this should've been a field? See below)

Reply:
I think this should be an object: https://github.com/facebook/react/blob/9491dee79586d21c115cd4d5986c81b7d88d2b3f/packages/react/src/ReactBaseClasses.js#L26